regression/pages/common/utils.py

```diff
@@ -0,0 +1,526 @@
+# -*- coding: utf-8 -*-
+
+"""
+Different utilities to be used in tests
+"""
+
+import functools
```

Comment:
The order of imports doesn't make sense to me.  You should read through and use [what we recommend](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#imports-order) on readthedocs.